.taskcluster.yml

```diff
@@ -23,14 +23,8 @@ tasks:
           owner: &task_owner ${event.pusher.name}@users.noreply.github.com
           source: &task_source ${event.compare}
         scopes:
-          - "queue:scheduler-id:taskcluster-github"
-
           # Granted to role "repo:github.com/servo/servo:branch:*"
-          - "queue:create-task:highest:aws-provisioner-v1/servo-*"
-          - "queue:create-task:highest:proj-servo/*"
-          - "queue:route:index.project.servo.servo.*"
-          - "docker-worker:cache:servo-*"
-          - "docker-worker:capability:privileged"
+          - "assume:project:servo:decision-task/trusted"
```

Comment:
where can one edit these?

Reply:
In https://tools.taskcluster.net/auth/roles/project%3Aservo%3Adecision-task%2Ftrusted, as document in the edits to `etc/taskcluster/README.md` in this PR.